arc/src/main.c

```diff
@@ -1051,27 +1061,29 @@ static void handle_pme(struct zjs_ipm_message* msg)
         break;
     case TYPE_PME_READ_NEURON:
         DBG_PRINT("read: neuron id=%d\n", msg->data.pme.neuron_id);
-        if (msg->data.pme.neuron_id < 1 || msg->data.pme.neuron_id > 128) {
+        if (msg->data.pme.neuron_id < 1 ||
+            msg->data.pme.neuron_id > 128) {
             ERR_PRINT("invalid neuron id, must be between 1 and 128\n");
             ipm_send_error(msg, ERROR_IPM_INVALID_PARAMETER);
             return;
         }
 
-        neuron_data_t data;
         curie_pme_read_neuron(msg->data.pme.neuron_id, &data);
         msg->data.pme.category = data.category;
         msg->data.pme.n_context = data.context;
         msg->data.pme.aif = data.aif;
         msg->data.pme.min_if = data.min_if;
         memcpy(msg->data.pme.vector, data.vector, sizeof(data.vector));
-        DBG_PRINT("neuron: id=%d, CTX=%d, AIF=%d MIF=%d, cat=%d\n", data.context & NCR_ID,
-                                                                    data.conext & NCR_CONTEXT,
-                                                                    data.aif,
-                                                                    data.min_if
-                                                                    data.category);
+        DBG_PRINT("neuron: id=%d, CTX=%d, AIF=%d MIF=%d, cat=%d\n",
+                  data.context & NCR_ID,
+                  data.context & NCR_CONTEXT,
+                  data.aif,
+                  data.min_if,
+                  data.category);
```

Comment:
I'd pack these as tight as possible in 80 chars, e.g. use 2-3 lines. Especially for a debug print, there's no need for special clarity in arguments - it's not real code you care about reading carefully, so it's just taking up more screen space.